app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
+
```

Comment:
- [ ] Please kindly indicate the foreign key attribute id for the `has_many` relationships. See the example below and implement it for others.

`has_many :posts, foreign_key: 'author_id'`